cookie-banner-rules-list.json

```diff
@@ -5580,6 +5580,390 @@
+      "click": {
+        "presence": "#Footer_Cookie",
+        "optOut": "#Footer_butCancel",
+        "optIn": "#Footer_butAccept"
+      }
+    },
+    {
+      "id": "c79fc6da-0143-46a0-abfc-debbd4d05f4b",
+      "domains": ["gainward.com", "palit.com"],
+      "click": {
+        "presence": "#pp_info",
+        "optOut": "[value=\"Reject\"]",
+        "optIn": "[value=\"Accept\"]"
+      }
+    },
+    {
+      "id": "6c9b123a-ec42-4128-91a2-a4cdd65059bc",
+      "domains": ["seasonic.com"],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "cookiepermission",
+            "value": "yes"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "cookiepermission",
+            "value": "no"
+          }
+        ]
+      },
+      "click": {
+        "presence": "#cp-overlay",
+        "optOut": "#cp-no",
+        "optIn": "#cp-yes"
+      }
+    },
+    {
+      "id": "d01204bd-8a94-4e6e-8ce4-d155b0681053",
+      "domains": ["synology.cn", "synology.com"],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "syno_confirm_v4_answer",
+            "value": "{\"necessary\":true,\"performance\":true,\"functionality\":true,\"targeting\":true}"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "syno_confirm_v4_answer",
+            "value": "{\"necessary\":true,\"performance\":false,\"functionality\":false,\"targeting\":false}"
+          }
+        ]
+      },
+      "click": {
+        "presence": ".syno_cookie_element",
+        "optIn": ".btn_accept"
+      }
+    },
+    {
+      "id": "9034f6e4-09ca-42a2-a8da-4f65968b8b36",
+      "domains": ["teamgroupinc.com"],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "privacy",
+            "value": "{\"necessary\":\"ok\",\"functional\":\"ok\",\"marketing\":\"ok\"}"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "privacy",
+            "value": "{\"necessary\":\"ok\"}"
+          }
+        ]
+      },
+      "click": {
+        "presence": ".privacyArea",
+        "optIn": ".acceptAll"
+      }
+    },
+    {
+      "id": "48e9b863-c642-4a7a-9ee5-c085d337233e",
+      "domains": ["tp-link.com"],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "tp_privacy_base",
+            "value": "1"
+          },
+          {
+            "name": "tp_privacy_marketing",
+            "value": "1"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "tp_privacy_base",
+            "value": "1"
+          }
+        ]
+      },
+      "click": {
+        "presence": "#tp-cookie",
+        "optIn": ".tp-cookie-accept-all"
+      }
+    },
+    {
+      "id": "5e0387bb-1f19-4f61-b587-49d995a691c9",
+      "domains": ["transcend-info.com"],
+      "click": {
+        "presence": "#legal_notice",
+        "optOut": "[onclick*=\"cookie_Disagree()\"]",
+        "optIn": "[onclick*=\"cookie_Agree()\"]"
```

Comment:
This seems quite fragile, but I don't think there's a more reliable way to target these two elements, unfortunately.